docs/src/main/sphinx/functions/json.rst

```diff
@@ -2,6 +2,1128 @@
 JSON functions and operators
 ============================
 
+The SQL standard describes functions and operators to process JSON data. They
+allow you to access JSON data according to its structure, generate JSON data,
+and store it persistently in SQL tables.
+
+Importantly, the SQL standard imposes that there be no dedicated data type to
```

Comment:
that there are no

Reply:
What's the grammar rule here? I thought I should use an infinitive. If not the infinitive, then singular "is" sounds better to me...

Reply:
"there be no" ... is very passive and formal like in a spec ... not suitable and hard to understand for many normal readers.. singular "is" is good